src/Analysis/GWASResultsV2/Views/Home/Home.jsx

```diff
@@ -13,35 +13,26 @@ const Home = () => {
     const getWorkflows = await fetch(workflowsEndpoint);
     return getWorkflows.json();
   }
-  const GWASWorkflows = () => {
```

Comment:
It turns out if you use a function here to output the needed JSX, it causes a re-render when the states associated with the homeTable inputs change, which causes them to lose focus. Placing the function inline fixes that problem.